tests/test_available_tasks.py

```diff
@@ -0,0 +1,24 @@
+# This Source Code Form is subject to the terms of the Mozilla Public
+# License, v. 2.0. If a copy of the MPL was not distributed with this
+# file, You can obtain one at http://mozilla.org/MPL/2.0/.
+
+import pytest
+
+from pages.home import HomePage
+
+
+class TestAvailableTasks():
+
+    @pytest.mark.nondestructive
+    def test_available_tasks(self, base_url, selenium):
+        home_page = HomePage(base_url, selenium).open()
+        assert not home_page.is_user_logged_in
+
+        available_tasks_page = home_page.click_available_tasks()
+        assert available_tasks_page.is_available_tasks_list_visible
+        assert len(available_tasks_page.available_tasks) > 0
+
+        task = available_tasks_page.available_tasks[0]
+        task_name = task.name
+        task_details = task.click()
+        assert task_details.name == task_name
```

Comment:
@MonicaG Oops, I must have missed this on my previous reviews. We try to stick to the standard of assert *expected* == *actual* but in this line I think you're doing actual first and then expected. Could you correct that please?